lblod/app-meldingsplichtige-api
# Bundle new forms - "Aanduiding en eedaflegging van de aangewezen burgemeester" - "Strandconcessies - reddingsdiensten kustgemeenten" - "Melding onvolledigheid inzending eredienstbestuur" - "Opstart beroepsprocedure naar aanleiding van een beslissing"

# Description

DL-5669 - DL-5625 - DL-5643 - DL-5646 - DL-5665 - DL-5670

This PR is : 

- Adding new form Aanduiding en eedaflegging van de aangewezen burgemeester

- Adding new form Strandconcessies - reddingsdiensten kustgemeenten

- Adding new form Opstart beroepsprocedure naar aanleiding van een beslissing

- Adding new form Melding onvolledigheid inzending eredienstbestuur

- Adding new form Afschrift erkenningszoekende besturen

# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

:warning:  Before running the branch I suggest creating a backup as codelist is subject to change

1. Generate dist file and copy the forms to consuming apps
2. Make sure the following migration has run in Loket 
3. Log as Gemeente (All forms are visible by Gemeenten but you can chose other units based on the visibleBy rule) and Bestuur van de Eredienst for the form "Afschrift erkenningszoekende besturen"
4. Save the form, it should be saved without errors.
5. Send the form, it should be consumed by app-toezicht-abb/worship-decisions and not by app-public-decisions-database.
a. To consume the form you can follow the docs @ https://github.com/lblod/app-toezicht-abb#trigger-import-export-flow-from-app-digitaal-loket
 
# What to check

- Typos, and form structure  

# Links to other PR's

- Manage-submissions-form-tooling -> https://github.com/lblod/manage-submission-form-tooling/pull/44 & https://github.com/lblod/manage-submission-form-tooling/pull/45 & https://github.com/lblod/manage-submission-form-tooling/pull/46
- App-digitaal-loket -> https://github.com/lblod/app-digitaal-loket/pull/528
- App-toezicht-ABB -> https://github.com/lblod/app-toezicht-abb/pull/38
- App-public-decisions-database -> https://github.com/lblod/app-public-decisions-database/pull/25
- App-worship-decisions-database -> https://github.com/lblod/app-worship-decisions-database/pull/59
- Prepare-submissions-for-export-service -> https://github.com/lblod/prepare-submissions-for-export-service/pull/12 & https://github.com/lblod/prepare-submissions-for-export-service/pull/13
- Worship-submissions-graph-dispatcher-service -> https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/18 & https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/19
- Enrich-submission-service -> https://github.com/lblod/enrich-submission-service/pull/18 & https://github.com/lblod/enrich-submission-service/pull/19

# Notes

-  drc restart migrations resource cache enrich-submission
